modules/nlu/src/backend/pipelines/entities/pattern_extractor.test.ts

```diff
@@ -263,5 +344,38 @@ I'm riding my mercedes-benz to the dealership then I will take my BM to buy an o
       expect(entities[3].meta.source).toEqual('one two tree')
       expect(entities[3].data.value).toEqual('one')
     })
+
+    test('fuzzy match should not be to loose', async () => {
+      const entityDef = {
+        id: '_',
+        name: 'bunchOfChars',
+        type: 'list',
+        fuzzy: true,
+        occurences: [
+          {
+            name: 'aescp',
```

Comment:
as we discussed, I know its nothing but you inverted 2 letters ==> " association des etudiants des cycles supérieurs de l'école polytechnique "